internal/helm/client.go

```diff
@@ -146,3 +146,13 @@ func (c *SynchronizedClient) Status(name string) (*helmrelease.Release, error) {
 	statusAction.ShowResources = true
 	return statusAction.Run(name)
 }
+
```

Comment:
Missing: PR updating the docs with the necessary explanations (reference and how-to)
how-to can be based on the explanation in this PR's descriptions on how to test
(could also be an extension of the existing `how to create a service`, as advanced section)

reference is changes to the existing service reference (customization)

Reply:
Sure, I will update/add the relevant docs.

Reply:
Ref: https://github.com/epinio/epinio/pull/2530